sdks/python/apache_beam/coders/coders_test_common.py

```diff
@@ -248,12 +249,13 @@ def test_memoizing_pickle_coder(self):
   @parameterized.expand([
       param(compat_version=None),
       param(compat_version="2.67.0"),
+      param(compat_version="2.68.0"),
```

Comment:
Could you please call out somewhere in a comment notable behavior differences in 2.67.0, 2.68.0 that are relevant for this test scenario?